Desktop/ReactNativeNavigator-start/W10/Profile.tsx

```diff
@@ -0,0 +1,307 @@
+import React, { useState, useEffect, useCallback } from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  Image,
+  FlatList,
+  ActivityIndicator,
+  Button,
+  TextInput,
+} from "react-native";
+import { StatusBar } from "expo-status-bar";
+import * as ImagePicker from "expo-image-picker";
+import {
+  useFocusEffect,
+  useNavigation,
+  useRoute,
+} from "@react-navigation/native";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { StackNavigationProp } from "@react-navigation/stack";
+import Modal from "react-native-modal"; // Импортируем библиотеку для модального окна
+
+type YourStackParamList = {
+  Home: undefined;
+  Profile: undefined;
+};
+
+type ProfileScreenNavigationProp = StackNavigationProp<
+  YourStackParamList,
+  "Profile"
+>;
+
+interface AnimeData {
+  url: string;
+  title: string;
+  images: {
+    jpg: {
+      image_url: string;
+    };
+  };
+}
+
+interface FavoriteItemProps {
+  url: string;
+  navigation: any;
+  onRemove: (url: string) => void;
+}
+
+const FavoriteItem: React.FC<FavoriteItemProps> = ({
```

Comment:
You should only have one component per file. FavoriteItem should be its own file and Profile should be its own file.